atomic_reactor/plugins/post_koji_upload.py

```diff
@@ -473,7 +473,7 @@ def add_log_type(output, arch):
 
         digests = self.get_digests()
         repositories = self.get_repositories(digests)
-        tags = set(image.tag for image in self.workflow.tag_conf.primary_images)
+        tags = set(image.tag for image in self.workflow.tag_conf.images)
```

Comment:
There are literally no primary tags for worker builds any more, so the metadata included "tags: []". Hmm, this should go in a separate commit I guess.